LSP-intelephense.sublime-settings

```diff
@@ -1,24 +1,26 @@
 {
-	"languages": [
-		{
-			"languageId": "php",
-			"scopes": ["source.php"],
-			"syntaxes": [
-				"Packages/PHP/PHP.sublime-syntax",
-			],
+	"intelephense": {
```

Comment:
But why? The plugin code doesn't read it from that key as far as I can see.

Reply:
I've looked at the issue and I get what you were trying to do here but I think you meant to put that key within the `settings` object.

Reply:
Ah, yes!